packages/@angular/cli/package.json

```diff
@@ -26,10 +26,6 @@
   },
   "homepage": "https://github.com/angular/angular-cli",
   "dependencies": {
-    "@angular/compiler": "^2.3.1",
-    "@angular/compiler-cli": "^2.3.1",
-    "@angular/core": "^2.3.1",
```

Comment:
If you're dropping `@angular/core` shouldn't you also drop the recently added `rxjs` dependency?